sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubTests.cs

```diff
@@ -27,7 +27,8 @@ public void GetStaticBindingContract_ReturnsExpectedValue()
             var strategy = new EventHubTriggerBindingStrategy();
             var contract = strategy.GetBindingContract();
 
-            Assert.AreEqual(7, contract.Count);
+            Assert.AreEqual(8, contract.Count);
```

Comment:
We should also add a live test in [EventHubEndToEndTests](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubEndToEndTests.cs) that uses the binding and the new property.

Reply:
Other than this end-to-end test tweak (which I'm not confident I could validate myself due to unfamiliarity with the process), I think this is good to merge now? Is it possible for one of the MS team tweak and validate this end-to-end test?

Reply:
Sure, I can take ownership for adding a test and merging this PR. Thanks @creyke!